setup.py

```diff
@@ -147,7 +147,7 @@ def run(self):
 
 setup_params = dict(
 	name="ufoLib",
-	version="2.2.1",
+	version="2.2.2",
```

Comment:
the version number should not be modified manually, but only using the `python setup.py bump_version` subcommand (this bumps the version and commits the change). The command needs to be called immediately _after_ `python setup.py release` to open up a new release cycle, but it seems we forgot to do that. Please revert this change, run `python setup.py bump_version`, then push the commit.